Automattic/wp-calypso
# Offer Reset: Upsell page and UpgradeNudge tweeks

#### Changes proposed in this Pull Request

`UpgradeNudge` changes:
* Make the feature checkmarks of the `UpgradeNudge` component pink.

`UpsellPage` changes:
* Surround the product card in a white box and set a max-width.
* Show the upsell product list of features.
* Add a back button bar at the top of the page (the same present in the `DetailsPage`).

#### Testing instructions

* Run this PR with the Offer Reset AB experiment enabled.

`UpgradeNudge` changes
* With a Jetpack site that has Jetpack Security **Daily**, visit the Plans page.
* Verify that the `UpgradeNudge` component feature checkmarks are pink (`#C9356E`).

`UpsellPage` changes:
* With a Jetpack site visit the Plans page.
* Visit the last `UpsellPage` of the process by trying to purchase Jetpack Backup or Jetpack Scan.
* Verify that the page looks like the ticket/design.
* Verify that the back button works as expected.
* Verify that the upsell product feature list is visible.
* Make sure to test these changes on different screen sizes to confirm that it scales well.

Fixes 1169247016322522-as-1190147348789665

#### Demo
`UpgradeNudge`
![image](https://user-images.githubusercontent.com/3418513/91092900-0df17200-e62f-11ea-9662-fb05196b2ee7.png)

`UpsellPage`
![image](https://user-images.githubusercontent.com/3418513/91093379-b99ac200-e62f-11ea-9709-8d4a4e8b900b.png)
